README.md

```diff
@@ -18,6 +19,16 @@ Implementation of MCP server for [Plainly](https://www.plainlyvideos.com/) in No
 
 ## ▶️ How to run (local installation)
 
+### Installing via Smithery
```

Comment:
* I would add those commands for specific clients as well
* Would also add a section in the help center once this is merged